injected/src/content-feature.js

```diff
@@ -198,8 +235,10 @@ export default class ContentFeature {
 
     /**
      * Given a config key, interpret the value as a list of domain overrides, and return the elements that match the current page
+     * Consider using patchSettings instead as per `getFeatureSetting`.
```

Comment:
Hm, this is confusing, did you mean "consider using `getFeatureSetting` instead of this method"?